modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -200,7 +202,18 @@ function MultipleChoice(props) {
     </Typography>
     );
 
-  const answers = selection.map(item => item[VALUE_POS] === GHOST_SENTINEL ? [item[LABEL_POS], item[LABEL_POS]] : item);
+  //Remove the ["", ""]
+  const tmp_answers = selection.map(item => item[VALUE_POS] === GHOST_SENTINEL ? [item[LABEL_POS], item[LABEL_POS]] : item);
+  var answers = [];
+  if (tmp_answers.length < 2) {
+    answers = tmp_answers.slice();
+  } else {
+    for (var i = 0; i < tmp_answers.length; i++) {
+      if (tmp_answers[i][0] != "") {
+        answers.push(tmp_answers[i]);
+      }
+    }
+  }
```

Comment:
Couldn't this be condenset with:

```suggestion
  const answers = selection.map(item => item[VALUE_POS] === GHOST_SENTINEL ? [item[LABEL_POS], item[LABEL_POS]] : item).filter(item => item[0] !== '');
```

Reply:
Almost...but your suggestion does not consider the case where `tmp_answers.length < 2`. I have still refactored this code down to two lines in commit `3100dc0`. I have done a similar refactoring for the `removeAllNull` method in commit `c45b1ff`